src/main/java/com/medbay/service/EmployeeService.java

```diff
@@ -0,0 +1,61 @@
+package com.medbay.service;
+
+import com.medbay.domain.Employee;
+import com.medbay.domain.User;
+import com.medbay.domain.enums.Role;
+import com.medbay.domain.request.CreateEmployeeRequest;
+import com.medbay.repository.EmployeeRepository;
+import com.medbay.repository.PatientRepository;
+import com.medbay.repository.TherapyRepository;
+import com.medbay.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+@Service
+@RequiredArgsConstructor
+public class EmployeeService {
+
+    private final EmployeeService employeeService;
+    private final PatientRepository patientRepository;
+    private final TherapyRepository therapyRepository;
+    private final EmployeeRepository employeeRepository;
+    private final UserRepository userRepository;
+
+    public ResponseEntity<List<Employee>> getEmployees(){
+        List<Employee> employees = employeeRepository.findAll();
+        return ResponseEntity.ok(employees);
+    }
+
+    public ResponseEntity<Void> createEmployee(CreateEmployeeRequest request){
+
+        Optional<User> user = userRepository.findByEmail(request.getEmail());
+        if(user.isEmpty()){
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
+        }
+
+        Employee employee = Employee.builder()
+                .firstName(request.getFirstName())
+                .lastName(request.getLastName())
+                .email(request.getEmail())
+                .password(request.getPassword())
```

Comment:
![image](https://github.com/Project-MedBay/backend/assets/111597257/d4a9b30a-5b3d-4e28-9b19-1f179fa07912)
pogledaj kako se ovdje stavlja sifra tako i ti napravi (samo to gledaj, drugo je nevazno)